cmd/mesh-bootstrap/main.go

```diff
@@ -57,8 +57,27 @@ type Config struct {
 	// P2PAdvertiseAddrs is a comma separated list of libp2p multiaddresses which the
 	// bootstrap node will advertise to peers.
 	P2PAdvertiseAddrs string `envvar:"P2P_ADVERTISE_ADDRS"`
-	// DataDir is the directory used for storing data.
+	// DataStoreType is the data store which will be used to store DHT data
+	// for the bootstrap node.
+	// DataStoreType can be either: leveldb or postgres
+	DataStoreType string `envvar:"DATA_STORE_TYPE" default:"leveldb"`
+	// DataDir is the directory used for storing data when using leveldb as data store type.
 	DataDir string `envvar:"DATA_DIR" default:"0x_mesh"`
+	// DataDBHost is the database host used to connect to the database when
+	// using postgres as data store type.
+	DataDBHost string `envvar:"DATA_DB_HOST" default:"localhost"`
```

Comment:
Can we organize all of these into a [connection URL/connection string](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING) to avoid having so many different env vars? Something like `postgresql://postgres@localhost:5432/datastore`.

Reply:
Maybe we could provide both options for Datastore creation in the `sql-datastore`? My idea to use seperate connection settings from a "deployment" and configuration management perspective is that:
- the password can be stored seperately (like in Kubernetes `secret` objects)
- the configuration is reusable / composable, for example if we have multiple bootstrap nodes the `DB_HOST`, `DB_PORT` will be common, however the `DB_USER` and `DB_PASSWORD` will be specific to each instance. And in the case of a db migration to a new host we wouldn't need to update different connection strings.

Reply:
If it were me managing deployments, I would much prefer having one environment variable instead of many. Maybe it's subjective though. Do whatever you think is best.

I don't like having both `DB_URL` and `DB_HOST`, `DB_PORT`, `DB_PASSWORD`, etc.